WooCommerce/Classes/ViewRelated/Coupons/CouponListViewController.swift

```diff
@@ -1,17 +1,17 @@
 import UIKit
 import WordPressUI
 
-final class CouponManagementViewController: UIViewController {
+final class CouponListViewController: UIViewController {
     @IBOutlet private weak var tableView: UITableView!
-    private var viewModel: CouponManagementListViewModel!
+    private var viewModel: CouponListViewModel!
```

Comment:
nit: this is unrelated to this PR, but I wonder if it'd be safer and look nicer to have it as a constant

```suggestion
    private let viewModel: CouponListViewModel
```

Reply:
Great catch! Since the `CouponListViewModel`'s initializer requires 2 closures as parameters that are taken from the controller, I believed this was set up intentionally 😅. I think I'll have to update the state management of the view controller to address this, so this will be handled in a separate PR. Thank you!